recipes/pypandoc/meta.yaml

```diff
@@ -0,0 +1,38 @@
+{% set version = "1.1.3" %}
+package:
+  name: pypandoc
+  version: {{ version }}
+
+source:
+  fn: pypandoc-{{ version }}.zip
+  url: https://github.com/bebraw/pypandoc/archive/v{{ version }}.zip
+  sha1: a20637259009c89b8f9b22342d4dc6dd5b516d83
+
+build:
+  number: 0
+  script: python setup.py install --single-version-externally-managed --record record.txt
+
+requirements:
+  build:
+    - python
+    - setuptools
+    - pandoc
+  run:
+    - python
+    - pandoc
+
+test:
+  imports:
+    - pypandoc
+  commands:
+    - python -c "import pypandoc; print(pypandoc.get_pandoc_version())"
```

Comment:
It's in the github zip, but I think we don't include it in the pypi upload. It's small enough to be run each time the package is uploaded... Let's see ...